index.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Emeka's Portfolio</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <header class="header1">
+
+        <h3 class="logo"><a href="#">My Logo</a></h3>
+
+        <nav>
+
+            <img src="./img/hamburger.png" alt="hamburger_image">
+
+        </nav>
+
+    </header>
+
+    <section class="section1">
+
+        <div class="intro">
+
+            <h1>I'm Emeka Glad to see you!</h1>
+
+            <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesistate to contact me. </p>
+
+        </div>
+
+        <div class="socialmedia_connect">
+
+            <h5>LET'S CONNECT</h5>
+
+            <ul>
+
+                <li><a href="#"><img src="./img/angular.jpg" alt="angular_image"></a></li>
+                <li><a href="#"><img src="./img/github.jpg" alt="github_image"></a></li>
+                <li><a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a></li>
+                <li><a href="#"><img src="./img/mee.jpg" alt="mee_image"></a></li>
+                <li><a href="#"><img src="./img//twitter.jpg" alt="twitter_image"></a></li>
+
+            </ul>
+
+        </div>
+
+    </section>
```

Comment:
- [ ] Impressive job on your readme @EmekaEmmanuel, One thing left you the background image is not as close as possible to the Figma design.
- [ ] Also, Your `I'm Emeka Glad to see you!` text is a bit lighter compared with the Figma design. Please update this to give recruiters confidence you can work with UX/UI designers.
- [Optional] A bit more spacing between your `social media icons` would be great. 
### Figma design 
![Capture2](https://user-images.githubusercontent.com/69077061/205231994-264c95ff-1394-4492-a795-98ce2c3d74f3.PNG)

### Your design 
![Capture](https://user-images.githubusercontent.com/69077061/205232012-9d530cec-2625-4d82-b57d-62526cdbc4b6.PNG)